src/api.rs

```diff
@@ -501,17 +501,18 @@ impl Api {
         self.get("/")?.convert()
     }
 
-    /// Lists all the release file for the given `release`.
-    pub fn list_release_files(
+    /// Lists release files for the given `release`.
+    fn _list_release_files(
```

Comment:
I haven’t seen `_` prefixed fns in rust yet. You can just name this `list_release_files_by_checksum` as you are just straight up passing every argument through :-)

Reply:
Lol, you're right. That didn't even occur to me ^^

Reply:
Welp, true that. Will patch that :3

Reply:
https://github.com/getsentry/sentry-cli/pull/1457